<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcessMonitor</class>
 <widget class="QDialog" name="ProcessMonitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Process Monitor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="4">
    <widget class="QTextBrowser" name="textBrowser"/>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pbInterrupt">
     <property name="text">
      <string>&amp;Interrupt</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pbViewLog">
     <property name="text">
      <string>View &amp;log</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>169</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="pbClose">
     <property name="text">
      <string>&amp;Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbClose</sender>
   <signal>clicked()</signal>
   <receiver>ProcessMonitor</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>497</y>
    </hint>
    <hint type="destinationlabel">
     <x>446</x>
     <y>467</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
